### PR TITLE
Enhancement/9373 extend callout

### PR DESCRIPTION
**Pull request by zutigrm**
## Summary

Addresses issue:

- #9373

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 5.2 and PHP 7.4.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.
- [ ] Ensure there are no unexpected significant changes to file sizes.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/js/components/KeyMetrics/ConversionReportingNotificationCTAWidget.test.js [hunk 1]
```diff
@@ -67,6 +72,8 @@ describe( 'ConversionReportingNotificationCTAWidget', () => {
 	);
 
 	beforeEach( () => {
+		enabledFeatures.add( 'conversionReporting' );
```
Comment (zutigrm) [inline assets/js/components/KeyMetrics/ConversionReportingNotificationCTAWidget.test.js:75]: Since this component will surface only when feature flag is enabled, moved the inclusion of feature flag here, instead of it being added in individual tests

File: assets/js/components/KeyMetrics/ConversionReportingNotificationCTAWidget.test.js [hunk 2]
```diff
@@ -383,7 +384,7 @@ describe( 'ConversionReportingNotificationCTAWidget', () => {
 			expect( container ).toBeEmptyDOMElement();
 		} );
 
-		it( 'Does not render if new events would suggest metrics the user has already selected', async () => {
+		it( 'does not render if new events would suggest metrics the user has already selected', async () => {
```
Comment (zutigrm) [inline assets/js/components/KeyMetrics/ConversionReportingNotificationCTAWidget.test.js:387]: Changed the capitalisation of few existing tests for consistency

File: assets/sass/components/key-metrics/_googlesitekit-key-metrics-conversion-reporting-subtle-notification.scss [hunk 3]
```diff
@@ -43,6 +43,7 @@
 	}
```
Comment (zutigrm) [inline assets/sass/components/key-metrics/_googlesitekit-key-metrics-conversion-reporting-subtle-notification.scss:43]: SMall style adjustment purely to cover the edge case when both new events and lost events callouts are shown

File: assets/js/components/KeyMetrics/ConversionReportingNotificationCTAWidget.test.js [hunk 4]
```diff
@@ -579,4 +580,415 @@ describe( 'ConversionReportingNotificationCTAWidget', () => {
...
+				);
+				await waitForRegistry();
+
+				expect(
+					getByRole( 'button', { name: 'View metrics' } )
+				).toBeInTheDocument();
+			} );
+		} );
+
+		describe( 'user with manualy selected metrics', () => {
```
Comment (tofumatt) [inline assets/js/components/KeyMetrics/ConversionReportingNotificationCTAWidget.test.js:853]:
```suggestion
		describe( 'user with manually-selected metrics', () => {
```

**Review by tofumatt — CHANGES_REQUESTED** [review]
The code here looks good, just one very minor change I'll make.

But the QA Brief here is hard-to-follow as it has large paragraphs that aren't clear instructions and include notes/info.

It's great to leave notes in the QA Brief, but can you separate out the instructions on how to QA the issue from any notes about various features/QA tasks?

**Comment by zutigrm** [comment]
Thanks @tofumatt I updated the QAB

**Review by tofumatt — APPROVED** [review]
